common/aws/dynamodb/client.go

```diff
@@ -156,6 +157,35 @@ func (c *Client) UpdateItem(ctx context.Context, tableName string, key Key, item
 	return resp.Attributes, err
 }
 
+func (c *Client) IncrementBy(ctx context.Context, tableName string, key Key, itemKey string, itemValue uint64) (Item, error) {
```

Comment:
nit: `attr string, value uint64`? 
(`itemValue` sounds like it will be assigned this new value)